### PR TITLE
Updated list with MongoDB list

### PR DESCRIPTION
**Pull request by abisheksriram**
## What does this PR do?
Added MongoDB resources

### Description
This pull request adds MongoDB free E-Book

### Why is this valuable (or not)?
This resource title is not available in the repository. It is actually a emerging one now.

### How do we know it's really free?
It's under Attribution-NonCommercial 3.0 Unported license. so it is free to read.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a book

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1457,6 +1458,11 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Modelica by Example](http://book.xogeny.com)
 
 
+### MongoDB
+
+* [The Little MongoDB book](https://www.openmymind.net/mongodb.pdf)  (PDF)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:1463]: Add authoring attributions to satisfy CC license is a must

**Review by davorpa — CHANGES_REQUESTED** [review]
It's a good moment to make some changes. When done, request for a rewiew again using GitHub :arrows_clockwise: button.

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (abisheksriram) [reply books/free-programming-books-langs.md:1463]: sure, will make necessary changes

Comment (davorpa) [reply books/free-programming-books-langs.md:1463]: Already listed. Check and update resource metadata

https://github.com/EbookFoundation/free-programming-books/blob/83beecd381eabcdf8e54782a07358792e31b3644/books/free-programming-books-langs.md#L1520

**Comment by eshellman** [comment]
Duplicate. MongoDB is under NoSQL, maybe its time to make subsections?

**Pull request closed without merging**